### PR TITLE
commented

### PR DESCRIPTION
**Pull request by hasft**

### CODE REVIEW

File: apps/ufjtbot/src/github/pullRequest/closed.ts [hunk 1]
```diff
@@ -32,13 +32,14 @@ export default async function closed({ payload }: Context<'pull_request.closed'>
           title: title,
           state: merged ? 'merged' : 'closed',
           user: sender.login,
-          branch: base.ref
+          branch: base.ref,
+          number: pull_request.number
```
Comment (buljagung) [inline apps/ufjtbot/src/github/pullRequest/closed.ts:36]: lgtm

**Comment by buljagung** [comment]
lgtm

**Review by buljagung — COMMENTED** [review]
lgtm

File: .changeset/brown-foxes-perform.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@ufjt-poc/bot": patch
```
Comment (hasft) [inline .changeset/brown-foxes-perform.md:2]: dodo

Comment (hasft) [reply .changeset/brown-foxes-perform.md:2]: kakaka

Comment (hasft) [reply .changeset/brown-foxes-perform.md:2]: kambing

File: apps/ufjtbot/src/github/pullRequest/closed.ts (hunk 1 above)
Comment (hasft) [reply apps/ufjtbot/src/github/pullRequest/closed.ts:36]: lambo

Comment (hasft) [reply apps/ufjtbot/src/github/pullRequest/closed.ts:36]: ayam dan lapeh

Comment (hasft) [reply apps/ufjtbot/src/github/pullRequest/closed.ts:36]: mbeling

Comment (hasft) [reply apps/ufjtbot/src/github/pullRequest/closed.ts:36]: bas gitar

Comment (hasft) [reply apps/ufjtbot/src/github/pullRequest/closed.ts:36]: karikatural

**Pull request closed without merging**